docs/src/guide/editor-support/index.md

```diff
@@ -1,10 +1,61 @@
 # {#guide-editor-support} IDE/Text Editor Support
 
-There are a number of text editor plugins available to provide a
-better development experience for Reach. These plugins provide syntax
-highlighting, build commands, key mappings, snippets, and more. The
-following plugins are availabe:
+There are a number of text editor plugins available to provide a better development experience for Reach.
+These plugins provide syntax highlighting, build commands, key mappings, snippets, and more.
+The following plugins are available:
 
 + Visual Studio Code: [Reach IDE](https://marketplace.visualstudio.com/items?itemName=reachsh.reach-ide)
 + Atom: [Reach Language Support](https://atom.io/packages/language-reach), [Reach IDE](https://atom.io/packages/reach-ide)
-+ Sublime Text: [Reach IDE](https://github.com/chrisnevers/reach-ide-sublime)
\ No newline at end of file
++ Sublime Text: [Reach IDE](https://github.com/chrisnevers/reach-ide-sublime)
+
+We highly recommend using Visual Studio Code IDE, it is the most advanced `Reach` extension.
+
+# {#guide-install-VSCode} Install VSCode
+
+Visual Studio Code (VSCode) enables users to both write and compile `Reach` code.
+With the `Reach IDE` installed, VSCode compiles automatically in the background and alerts users of coding errors in seconds.
```

Comment:
```suggestion
With the `Reach IDE` VSCode extension installed, VSCode compiles automatically in the background and alerts users of coding errors in seconds when the necessary prerequisites, like Docker, are installed and running.
```

I prefer being as explicit as possible. `Reach IDE VSCode extension` emphasizes to readers, especially beginners, that the Reach IDE isn't its own, standalone program: it's an extension for Visual Studio Code.

We also want to make sure we set users' expectations appropriately. The extension's automatic, background compilation has "failed" on myself and others several times, when really it just failed to start because Docker wasn't running. I think it's really important to mention things like Docker being necessary for the automatic, background compilation to work.